get-started/welcome.qmd

```diff
@@ -0,0 +1,108 @@
+---
+title: "Welcome to Shiny"
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Shiny for Python is a powerful and beginner-friendly framework for building interactive web applications and dashboards, all in pure Python.
+Whether you’re a data scientist, analyst, or developer, Shiny makes it easy to create rich, interactive experiences without needing to learn JavaScript or front-end frameworks.
+
+## Batteries included: everything you need to build a dashboard {#batteries-included}
```

Comment:
Let's rewrite these headings to avoid the "Word: phrase" pattern. Stick with either the word or the phrase (or some mash-up of the two) depending on the scenario.

Reply:
:heavy_check_mark: Renamed to: 

- Why choose Shiny for Python
- Batteries included
- Reactivity
- Templates
- Extensibility
- Give it a try